guide/14-deep-learning/use_mistral_llm_for_text_classification_and_entity_recognition.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>To load a saved mistral model, use the <code>from_model</code> method from the <code>TextClassifier</code> class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>